platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblems.kt

```diff
@@ -182,7 +182,11 @@ class ConfigurationCacheProblems(
     }
 
     override fun onExecutionTimeProblem(problem: PropertyProblem) {
-        val severity = if (isWarningMode) ProblemSeverity.Deferred else ProblemSeverity.Interrupting
+        val severity = when {
+            isWarningMode -> ProblemSeverity.Deferred
+            cacheAction == SkipStore || degradationDecision.shouldDegrade -> ProblemSeverity.SuppressedSilently
```

Comment:
❓ Will it be visible in the report in the end?

Reply:
Yeah, that is the goal, the only difference is to have a weaker severity. Seemed the least impacting change that would fix the issue.